networking/ingress/configuring-default-certificate.adoc

```diff
@@ -0,0 +1,16 @@
+[id="configuring-default-certificate"]
+= Configuring the default certificate
+include::modules/common-attributes.adoc[]
+:context: configuring-default-certificate
+
+toc::[]
+
+The following sections describe how to create a re-encrypt route with a custom
+certificate, how to create an edge route with a custom certificate, and how to
+set a custom default certificate.
+
+include::modules/nw-ingress-creating-a-reencrypt-route-with-a-custom-certificate.adoc[leveloffset=+1]
+
+include::modules/nw-ingress-creating-an-edge-route-with-a-custom-certificate.adoc[leveloffset=+1]
+
+include::modules/nw-ingress-setting-a-custom-default-certificate.adoc[leveloffset=+1]
```

Comment:
The first two modules are more for developers (in v3, it would in the "Developer Guide") whereas the third is an administrative task (in v3, "Configuring Clusters" or possibly "Day Two"), but we are not making that distinction in the v4 documentation, right?

I would still put the `nw-ingress-setting-a-custom-default-certificate.adoc` first since it describes configuring the default, which pretty much any administrator will want to do, and the other two modules describe how to use custom certificates instead of the default.  Does that make sense?

Reply:
That makes sense. I rearranged the order, and added some context for admin versus dev tasks.